instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/DbClientCommonAttributesGetter.java

```diff
@@ -24,7 +24,9 @@ default String getDbSystem(REQUEST request) {
 
   @Deprecated
   @Nullable
-  String getUser(REQUEST request);
+  default String getUser(REQUEST request) {
```

Comment:
Since these are removed in the stable semconv we don't need to force users to implement them.